client/lib/credit-card-details/validation.js

```diff
@@ -54,6 +54,20 @@ function creditCardFieldRules() {
 	};
 }
 
+function parseExpiration(value) {
+	const [month, year] = value.split( '/' );
```

Comment:
We put spacing almost everywhere in JS code, similar to PHP. See [Spacing docs](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing).

So there are couple of places where you need to update code in this PR.

I also recommend installing git pre-commit hook. See [here](https://github.com/Automattic/wp-calypso/blob/56aa9fbc09166180ce15c9eeb6a3f56cf32f7e80/Makefile#L181) or simply run from the command line:
`make githooks-commit`